d3a_api_client/utils.py

```diff
@@ -271,3 +277,20 @@ def log_bid_offer_confirmation(message):
                          f"{round(energy, 2)} kWh at {price} cts/kWh")
     except Exception as e:
         logging.error(f"Logging bid/offer info failed.{e}")
+
+
+def get_simulation_config(simulation_id=simulation_id_from_env, domain_name=domain_name_from_env,
+                          websockets_domain_name=websocket_domain_name_from_env):
+    if os.environ.get('SIMULATION_CONFIG_FILE_PATH', "") != "":
```

Comment:
I have the feeling that this check can be problematic for some reasons:
- `.get` will return `None` by default, and the truth value of `""` is `False` by default;
- if `SIMULATION_CONFIG_FILE_PATH` has not been defined, it will be `None` (and not `""`) so the current checks won't notice this issue.

I'd suggest replacing it with:
```python
if os.environ.get('SIMULATION_CONFIG_FILE_PATH'):
```